### PR TITLE
Add tutorial test

### PR DESCRIPTION
**Pull request by Chris-Sigopt**
This automates the run tutorial and runs it as part of rot-check, to emulate first run user experience.

### CODE REVIEW

File: ci/tutorial/model.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+model = XGBClassifier( 
+  n_estimators=sigopt.params.n_estimators, 
+  use_label_encoder=False, 
+  eval_metric='logloss', 
+) 
+model.fit(Xtrain, ytrain) 
+pred = model.predict(X) 
+
+# Track the metric value and metric name for each Run 
+sigopt.log_metric("accuracy", sklearn.metrics.accuracy_score(pred, y))
```
Comment (tjs-intel) [inline ci/tutorial/model.py:28]: can you think of some way to keep this up to date with https://app.sigopt.com/docs/tutorial/experiment?

**Review by tjs-intel — APPROVED** [review]

File: ci/tutorial/model.py (hunk 1 above)
Comment (nickivance) [reply ci/tutorial/model.py:28]: We're going to move that tutorial page out of the codebase to gitbook soon.

Comment (nickivance) [reply ci/tutorial/model.py:28]: If we could reference a Gist, however, that's something that could be used in Gitbook.

Comment (Chris-Sigopt) [reply ci/tutorial/model.py:28]: That's a good idea, will take a look when we're on gitbook.

**Pull request merged**